### ISSUE TITLE
Add Facebook OpenGraph metadata tags

### ISSUE THREAD

**Issue by willynilly** [omeka/plugin-SocialBookmarking#4]
Add Facebook Open Graph metadata tags.
http://support.addthis.com/customer/portal/articles/381222-optimize-facebook-sharing#.UPCLj4njnVQ

**Comment by dtygel** [comment]
No news on this issue? It seems very important for making omeka shares look beautiful and consistent in facebook....

**Comment by anuragji** [comment]
That should be fairly simple to implement - I suppose you'd just need a hook that gets added to the head when browsing an item :)

**Comment by zerocrates** [comment]
Yes, this should be possible pretty simply with the [public_head](http://omeka.readthedocs.org/en/latest/Reference/hooks/public_head.html) hook. I don't know if it makes sense to have that be part of Social Bookmarking or as its own plugin, but it would make some sense here, I suppose.
